utils/validate.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Validate
+  def valid?(*props)
```

Comment:
this module should contain helper methods like this:
```
def check_class(instance, klass)
  raise WrongClassError unless instance.instance_of? klass
end
```
Also, let's create `errors.rb` file that will contain error classed for our validations.
```
module Errors
  class WrongClassError < StandardError
    def initialize
      super('Error message')
    end
  end
end
```